action.yml

```diff
@@ -328,6 +332,23 @@ runs:
                   '\n[Full pipeline log](${{ steps.artifacts_url.outputs.url }}/${{ steps.sched_test.outputs.req_id }}/pipeline.log).'
           })
 
+    - name: Create github summary
+      if: ${{ inputs.create_github_summary == 'true' }}
+      shell: bash
+      run: |
+        if [ -z "${{ steps.final_state.outputs.INFRA_STATE }}" ]; then
+          infra_state=OK
+        else
+          infra_state=Failed
+        fi
+        if [ -z "$GITHUB_SUMMARY" ]; then
```

Comment:
What about if GITHUB_SUMMARY is set by the others, I would prefer to add there if a text is present in GITHUB_SUMMARY, then create a table.

Something like `grep "|OS|VERSION|" $GITHUB_SUMMARY`

Reply:
There is no way how we could show $OS and $VERSION in the TFaGA summary. $OS and $VERSION are solely sclorg-specific. 
We can however let user declare some string tio show in the summary via input variable.

We could show `pull_request_status_name` too, but its default value ( "Fedora") is problematic a bit as there is no way how to really tell if user set it or not.

In my opinion is sufficient to show `compose` and `arch`.

Reply:
I forgot that the other teams also used the action, so let's check `grep "Compose" $GITHUB_SUMMARY"` or "infrastructure" keyword. In order to avoid this, GITHUB_SUMMARY is set by the others

Reply:
So you suggest that the user should set the $GITHUB_SUMMARY by themself and then we get some information from it, drop it, and create new $GITHUB_SUMMARY?

Can you please explain further? I do not understand what you mean.

Reply:
Forgot about this. This table itself is created by us. I meant, if $GITHUB_SUMMARY is not empty, the user can add there before our action some text here then this condition and adding header is skipped.

Reply:
[eca59db](https://github.com/sclorg/testing-farm-as-github-action/pull/74/commits/eca59dbf243e3a6443a694a129312db971b024d5).